src/Configuration/test/PlaceholderBase.Test/PlaceholderResolverConfigurationExtensionsTest.cs

```diff
@@ -25,6 +25,7 @@ public void AddPlaceholderResolver_ThrowsIfConfigBuilderNull()
         }
 
         [Fact]
+        [Trait("Category", "SkipOnMacOS")]
         public void AddPlaceholderResolver_ThrowsIfConfigNull()
```

Comment:
This failed twice in a row on mac net5.0 build. But it wasn't a failure, the test runner crashed with this as most likely failure.  see: https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/results?buildId=5442&view=logs&j=57fb7e2e-87ff-5762-77a2-e1b9cb7a53c7&t=57c60dbc-0a83-516a-6c4f-11e50ea12df3

Reply:
Weird, it looks like the next build after that failed on `AddPlaceholderResolver_WithConfiguration_ReturnsNewConfiguration`. In all cases the stack trace of the exception mentions a `FileSystemWatcher`, so it probably wasn't either of these tests, but rather one of the (4) others in that file that deals with temp files.